census/README.md

```diff
@@ -1,313 +1,322 @@
+
+Using census data which contains data a person's age, education, marital status, 
+and occupation (the features), we will try to predict whether or not the person earns 
+more than 50,000 dollars a year (the target label). We will train a logistic regression 
+model that, given an individual's information, outputs a number between 0 and 1, this can 
+be interpreted as the probability that the individual has an annual income of over 50,000 dollars.
 
- * Training file is `adult.data.csv`
- * Evaluation file is `adult.test.csv`
+As a modeler and developer, think about how this data is used and the potential benefits and harm a model's predictions can cause. A model like this could reinforce societal biases and disparities. Is each feature relevant to the problem you want to solve or will it introduce bias? For more information, read about [ML fairness](https://developers.google.com/machine-learning/fairness-overview/).
+ 
+* **Disclaimer**
 
-### Disclaimer
 This dataset is provided by a third party. Google provides no representation,
 warranty, or other guarantees about the validity or any other aspects of this dataset.
 
-### Set Environment Variables
-Please run the export and copy statements first:
+* **Setup and test your GCP environment**
 
-```
-TRAIN_FILE=gs://cloud-samples-data/ml-engine/census/data/adult.data.csv
-EVAL_FILE=gs://cloud-samples-data/ml-engine/census/data/adult.test.csv
-```
+The best way to setup your GCP project is to use this section in this
+[tutorial](https://cloud.google.com/ml-engine/docs/tensorflow/getting-started-training-prediction#set-up-your-gcp-project).
 
-### \*Optional\* Use local training files.
+* **Environment setup:**
 
-Since TensorFlow - not the Cloud ML Engine - handles reading from GCS, you can run all commands below using these environment variables. However, if your network is slow or unreliable, you may want to download the files for local training.
+Virtual environments are strongly suggested, but not required. Installing this
+sample's dependencies in a new virtual environment allows you to run the sample
+without changing global python packages on your system.
 
-```
-mkdir census_data
+There are two options for the virtual environments:
+
+*   Install [Virtualenv](https://virtualenv.pypa.io/en/stable/) 
+    *   Create virtual environment `virtualenv myvirtualenv`
+    *   Activate env `source myvirtualenv/bin/activate`
+*   Install [Miniconda](https://conda.io/miniconda.html)
+    *   Create conda environment `conda create --name myvirtualenv python=2.7`
+    *   Activate env `source activate myvirtualenv`
 
-gsutil cp $TRAIN_FILE census_data/adult.data.csv
-gsutil cp $EVAL_FILE census_data/adult.test.csv
+* **Install dependencies**
 
-TRAIN_FILE=census_data/adult.data.csv
-EVAL_FILE=census_data/adult.test.csv
+Install the python dependencies. `pip install --upgrade -r requirements.txt`
+
+#
+
+* **How to satisfy Cloud ML Engine project structure requirements:**
```

Comment:
also this is not census-specific.  consider linking to the doc site instead of duplicating content here.

Reply:
Done